components/article/client/view.coffee

```diff
@@ -69,6 +73,15 @@ module.exports = class ArticleView extends Backbone.View
       @loadedCallouts = true
       @maybeFinishedLoading()
 
+  setupPartnerBreadcrumb: =>
+    new Partner(id: @article.get('partner_ids')?[0]).fetch
+      success: (partner) =>
+        new Profile(id: partner.get('default_profile_id')).fetch
```

Comment:
You may not need to fetch the profile here. It looks like all that's needed is [profile.href()](https://github.com/artsy/microgravity/blob/2ae509102792ef3561037772f0ac10ce8cb71f7b/models/profile.coffee#L15). The associated profile's id of a partner is `default_profile_id`, so you can maybe just use that for the href and save yourself a fetch.